include/common/core/arch_config.hpp

```diff
@@ -120,6 +120,7 @@ struct arch_attr_t<gpu_arch::Xe> {
     using mma_attr = mma_attr_t<gpu_arch::Xe>;
 
     static constexpr uint32_t max_wg_num = 64;
+    static constexpr uint32_t local_mem_size = 128 * 1024;
```

Comment:
are difference sku of PVC all equipped with same size of local mem?

Reply:
No idea. Actually I found that [DG2 should have 128k of SLM](https://www.intel.com/content/www/us/en/docs/oneapi/optimization-guide-gpu/2024-0/intel-xe-gpu-architecture.html#id-d12034e454) instead of 64K which i get from `device.get_info<info::device::local_mem_size>()`.